### PR TITLE
ldecicco's changes (fix legend overlap)

### PR DESCRIPTION
**Pull request by jiwalker-usgs**

### CODE REVIEW

**Comment by ldecicco-USGS** [comment]
FYI...it does not fix the legend overlap! If fixes the issue that if there wasn't data in the early part of the record, the timeline graphs would start at the first data point.

Here is what I'm proposing:
image

vs:

image

**Comment by jiwalker-usgs** [comment]
oh, sorry I wasn't paying attention to slack

File: scripts/visualize/portfolio_timeline.R [hunk 1]
```diff
@@ -1,101 +1,115 @@
...
+       viz.data$sessions,
+       type=line_type,
+       xlab="",ylab="", yaxt='n',xaxt='n',
+       xlim = range_days,
+       ylim = c(0, max(viz.data$sessions, na.rm = TRUE)),
+       frame.plot = FALSE)
+
+  last.tick <- tail(pretty(c(0, max(viz.data$sessions, na.rm = TRUE))),2)[1]
+
+  axis.Date(1, at=seq(from = range_days[1], to = range_days[2], by=day_breaks[type]),
```
Comment (ldecicco-USGS) [inline scripts/visualize/portfolio_timeline.R]: I did this wrong:
```
  pretty_days <- pretty(range_days) 
  axis.Date(1, at=seq(from = pretty_days[1],
                      to = pretty_days[length(pretty_days)],
                      by=day_breaks[type]),
            format=format_style[type])
```

**Comment by ldecicco-USGS** [comment]
And now with consistent x labels:

image

File: scripts/visualize/portfolio_timeline.R (hunk 1 above)
Comment (jiwalker-usgs) [inline scripts/visualize/portfolio_timeline.R]: ok, put this in

**Pull request merged**